apps/frontend/src/store/evaluations.ts

```diff
@@ -0,0 +1,87 @@
+import Store from '@/store/store';
+import {IEvaluation} from '@heimdall/interfaces';
+import axios from 'axios';
+import {
+  Action,
+  getModule,
+  Module,
+  Mutation,
+  VuexModule
+} from 'vuex-module-decorators';
+
+@Module({
+  namespaced: true,
+  dynamic: true,
+  store: Store,
+  name: 'EvaluationModule'
+})
+export class Evaluation extends VuexModule {
```

Comment:
I'm not sure on the naming of this, since profiles can be stored in the database in the same table as well. Although that may be a more systemic issue than just this one filename.

Reply:
That does seem more like a systemic issue, I think there should be a way to easily determine what's a profile, and what's an evaluation. Would an extra field in the data table work?

![image](https://user-images.githubusercontent.com/66680985/104632143-45f36300-566b-11eb-8bba-0546885d62e5.png)